cmake/IgnConfigureProject.cmake

```diff
@@ -55,40 +55,41 @@ macro(ign_configure_project)
   #============================================================================
   # Extract the designation
   #============================================================================
-  set(IGN_DESIGNATION ${PROJECT_NAME})
-  # Remove the leading "ignition-"
-  string(REGEX REPLACE "ignition-" "" IGN_DESIGNATION ${IGN_DESIGNATION})
+  set(GZ_DESIGNATION ${PROJECT_NAME})
```

Comment:
We've already made this change on `main`, with a tick-tock so that `IGN_DESIGNATION` still works. We should make sure `IGN_DESIGNATION` works here, otherwise we'll be breaking downstream projects.

Reply:
I've open PRs to all of our repositories that use IGN_DESIGNITION starting with versions for Acropolis.

Reply:
These variables could potentially be used by projects other than ours though, so we've been tick-tocking whatever seems reasonable. Tick-tocking here will also ensure that our stack isn't in a broken state once this PR is merged. Otherwise we'll need to rush to merge all downstream PRs and make new releases for all of Citadel and Fortress.

All we need to add here is this line:

https://github.com/gazebosim/gz-cmake/blob/8a437ab763d8c1c42da11b4cf8b8b169080433f4/cmake/GzConfigureProject.cmake#L110

Reply:
Actually we'll need a few other lines too, for variables like `IGN_DESIGNATION_CAP`

Reply:
can we just backport the tick-tock changes made on `main`?

Reply:
@scpeters , do you mean cherry-picking the commits or copying the changes? I'd be careful about cherry-picking, because there have been some hard-tocks and we don't want to introduce deprecation warnings on Citadel and Fortress.

Reply:
it would probably require manually copying since there were lots of changes in the same PR